tests/phpunit/CRM/Member/Form/MembershipTest.php

```diff
@@ -1319,7 +1319,7 @@ public function testTwoMembershipsViaPriceSetInBackendWithDiscount(): void {
     $discountedItems = 0;
     foreach ($lineItemResult['values'] as $lineItem) {
       $this->assertEquals(round($lineItem['line_total'] * .1, 2), $lineItem['tax_amount']);
-      if (CRM_Utils_String::startsWith($lineItem['label'], 'Long Haired Goat')) {
+      if (str_starts_with($lineItem['label'], 'Long Haired Goat')) {
```

Comment:
One can only guess why this is here... perhaps in answer to the question "What does our code smell like?"

Reply:
Fun story. I went to a restaurant and ordered spaghetti. After the waiter deposited the plate on the table, I picked up a fork and saw... it... a piece of string. Since string isn't yummy, I pulled it out - but it was quite long. It went over the edge of the plate, down the side of the table, through the walkway, past the kitchen door, over the vegetable prep station, and into the back office. Finally, in the back office, I could see that it started with a goat. So, yes, sometimes a string starts with a long haired goat.

Reply:
You should have brought it home to be our mascot. It could live in your living room and chew on the gold-plated record and other Civi paraphernalia.

Reply:
I think @eileenmcnaughton already gave it a good home.

Reply:
All long haired goats need CiviCRM memberships....

Reply:
Also - if you look down a couple more lines you will see there was a great discount